ngmix/tests/test_guessers.py

```diff
@@ -0,0 +1,164 @@
+import pytest
```

Comment:
We should improve the tests here. In particular, making sure the guessers are reproducible with the same RNG seed and that they produce different guesses each time is important.